README.md

```diff
@@ -1,46 +1,68 @@
 2. If your repository requires cloning credentials, configure them [here](https://github.com/moderneinc/enterprise-jenkins-ingest/blob/main/seed.groovy#L95) and [here](https://github.com/moderneinc/enterprise-jenkins-ingest/blob/main/seed.groovy#L108-L110).
 
-### If you want to publish ASTs to Nexus
-1. In `seed.groovy`, look for `artifactRepositoryType` [here](https://github.com/moderneinc/enterprise-jenkins-ingest/blob/main/seed.groovy#L1). Change the value to 'nexus'.
-2. In `publish-ast.groovy`, look for `repositoryRootUrl` [here](https://github.com/moderneinc/enterprise-jenkins-ingest/blob/main/publish-ast.groovy#L34). Modify that to point to the Nexus repository url that you want to publish ASTs to.
-3. On initial run, the `publish-ast.groovy` script execution step will fail due to in-process script approval being required. Approve the script at `{JENKINS_URL}/scriptApproval/`.
-4. Create credentials in jenkins with id 'nexus', of kind "Username with password" with credentials to use when publishing to Nexus. These credentials are bound [here](https://github.com/moderneinc/enterprise-jenkins-ingest/blob/main/seed.groovy#L112-L114).
+#### If you want to publish ASTs to Nexus
+1. In [seed.groovy](/seed.groovy), update the [artifactRepositoryType variable](https://github.com/moderneinc/enterprise-jenkins-ingest/blob/main/seed.groovy#L1) to have a value of `nexus`.
+2. In [publish-ast.groovy](/publish-ast.groovy), update [repositoryRootUrl](https://github.com/moderneinc/enterprise-jenkins-ingest/blob/main/publish-ast.groovy#L34) to point to the Nexus repository url that you want to publish ASTs to.
+    >**Note**: On the initial run, the `publish-ast.groovy` script execution step will fail due to in-process script approval being required. Approve the script at `{JENKINS_URL}/scriptApproval/`.
+3. Create credentials in Jenkins. The `id` should be `nexus` and the `type` should be `Username with password`. The credentials should be the ones you use to publish to Nexus. These credentials are bound [here](https://github.com/moderneinc/enterprise-jenkins-ingest/blob/main/seed.groovy#L112-L114).
+
+### Update ingestion time (optional)
+In [seed.groovy](/seed.groovy), there is a [triggers section](https://github.com/moderneinc/enterprise-jenkins-ingest/blob/main/seed.groovy#L103-L105) that specifies when the ingestion should happen. By default, it will run at around 4:00 am server time every day (Jenkins will pick what minute to run [based on load](https://stackoverflow.com/questions/26383778/spread-load-evenly-by-using-h-rather-than-5)).
+
+If you'd like to choose a different time or have it run more often, you'll want to update [this line](https://github.com/moderneinc/enterprise-jenkins-ingest/blob/main/seed.groovy#L103-L105).
```

Comment:
I guess in order to set up the on-commit hook i talked about earlier, we'd have to make changes to the template seed.groovy. Maybe we should create an issue for that.

Reply:
Do you mean create an issue for enabling that or teaching people how to do that or something else?

Reply:
I was suggesting we create an issue for enabling that.

Reply:
Created an issue: https://github.com/moderneinc/enterprise-jenkins-ingest/issues/8